### PR TITLE
Fix button offset when window resizes

### PR DESCRIPTION
**Pull request by JerryLingjieMei**
#345 with button offset determined by whether the rhombus technique point 1 is filled or not.

### CODE REVIEW

File: tasks/map/control/joystick.py [hunk 1]
```diff
@@ -13,6 +14,36 @@
...
+
+
+TECHNIQUE_POINT_LOOSE_0 = build_technique_points_loose(TECHNIQUE_POINT_0)
+TECHNIQUE_POINT_LOOSE_1 = build_technique_points_loose(TECHNIQUE_POINT_1)
+
+TECHNIQUE_POINT_STRICT_1 = build_technique_points_strict(TECHNIQUE_POINT_1)
+TECHNIQUE_POINT_STRICT_2 = build_technique_points_strict(TECHNIQUE_POINT_2)
+TECHNIQUE_POINT_STRICT_3 = build_technique_points_strict(TECHNIQUE_POINT_3)
+TECHNIQUE_POINT_STRICT_4 = build_technique_points_strict(TECHNIQUE_POINT_4)
+TECHNIQUE_POINT_STRICT_5 = build_technique_points_strict(TECHNIQUE_POINT_5)
```
Comment (LmeSzinc) [inline tasks/map/control/joystick.py:44]: Add `TECHINQUE_POINT_1.SEARCH.png` to enlarge search area, dont dynamically build buttons

File: module/base/base.py [hunk 2]
```diff
@@ -170,6 +170,7 @@ def image_crop(self, button, copy=True):
         Args:
             button(Button, tuple): Button instance or area tuple.
             copy:
+            offset:
```
Comment (LmeSzinc) [inline module/base/base.py:173]: Remove unused argments

**Comment by JerryLingjieMei** [comment]
Ready to merge, also fixed technique_pt_5 figure.

File: module/base/base.py [hunk 3]
```diff
@@ -187,6 +187,7 @@ def image_color_count(self, button, color, threshold=221, count=50):
             color (tuple): RGB.
             threshold: 255 means colors are the same, the lower the worse.
             count (int): Pixels count.
+            offset: if the button is offset
```
Comment (LmeSzinc) [inline module/base/base.py:190]: Remove unused argments

**Review by LmeSzinc — COMMENTED** [review]
Remove unused assets, keep only `TECHNIQUE_POINT_[012345]`, only `TECHNIQUE_POINT_[01]` have SEARCH attribute while the rest don't

**Pull request merged**